templates/terraform/examples/tpu_node_basic.tf.erb

```diff
@@ -9,8 +9,6 @@ data "google_tpu_tensorflow_versions" "available" {
 
 resource "google_tpu_node" "<%= ctx[:primary_resource_id] %>" {
   name = "<%= ctx[:vars]["node_name"] %>"
-  zone = "us-central1-b"
```

Comment:
One quick note- when zones other than `us-central1-a` have been selected, we generally want to explicitly run in that zone. TPUs either weren't available in `-a` or had recurring capacity issues causing us to use `-b`.

Reply:
I see. The zones were being overridden everywhere which is confusing.

Reply:
fixed